src/blocks/ImageTiles/ImageTiles.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import Ublock from '@src/components/Ublock/Ublock'
+
+export interface ImageTilesProps {
+    uBlockFull: boolean
+    mainData: ImageTilesMainProps
+    subData: ImageTilesSubProps[]
+}
+
+export interface ImageTilesMainProps {
+    image: HTMLImageElement
+    header: string
+    paragraph: string
+}
+
+export interface ImageTilesSubProps {
+    image: HTMLImageElement
+    header: string
+    paragraph: string
+}
+
+const ImageTiles: React.FC<ImageTilesProps> = ({
+    uBlockFull,
+    mainData,
+    subData,
+}): JSX.Element => {
+
+    return (
+        <Ublock full={uBlockFull}>
```

Comment:
As commented, Image Title is always full, so make it hard coded :)

Reply:
Done